webpack-build.config.js

```diff
@@ -27,12 +17,23 @@ webpackConfig.plugins.push(
 
   // Reference: http://webpack.github.io/docs/list-of-plugins.html#uglifyjsplugin
   // Minify all javascript, switch loaders to minimizing mode
-  new webpack.optimize.UglifyJsPlugin({
+  new webpack.optimize.UglifyJsPlugin( {
     compress: {
       warnings: false
     }
-  })
-
-);
+  } )
+];
+
+webpackConfig.plugins.push.apply( webpackConfig.plugins, webpackConfigPlugins );
```

Comment:
awesome! but I think we should stick with 4.x for the time being, because it's LTE